frontEnd/tickets-app/src/reducer/reducer.js

```diff
@@ -0,0 +1,20 @@
+const initialState = { user: undefined };
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case "LOGIN":
```

Comment:
create a separate file for action types and import action types from there. they should be constant.
the store info should be located in a `/store` folder. and it should be divided into slices.

f.e.
store/user: 
-/reducers.js
-/actions.js
-/selectors.js

etc.